Makefile

```diff
@@ -96,6 +100,8 @@ generate: bootstrap
 	popd \
 	)
 	GENERATE_TEST_DATA="true" go test ./pkg/agent...
+	@echo "compiling proto files in ./pkg/proto/ to pb.go files"
```

Comment:
do we actually need to do this on every `make generate`? I might lean towards keeping it as a separate target that we only run when changing those protos

Reply:
Makes sense. So we will run "make compile-proto-files" manually when we think needed